sass/components/_todo.scss

```diff
@@ -0,0 +1,13 @@
+.todo {
+  background-color: hsla(50, 50, 50, 0.3);
+  border: double hsla(50, 75, 50, 0.5) 6px;
+  border-radius: 10px;
+  padding: 4px;
+
+  h2.todo-header {
```

Comment:
The `todo-header` class seems unnecessary.

Reply:
I had to make it more specific to override unwanted huge margin styling from else where.

Reply:
I see, okay.